src/common/roll_renderer.js

```diff
@@ -595,6 +595,9 @@ class Beyond20RollRenderer {
                 }
             }
 
+            await this._roller.resolveRolls(request.name, all_rolls)
+            
+            //Moved after the new resolveRolls so it can access the roll results
```

Comment:
This comment is not needed, because it explains what this specific commit does, not the code. So reading the code back in a year, we see "moved after... " and we wonder "why is this saying moved?" 
A comment like "Handle Chaos Bolt after the rolls are resolved" would be nicer. Also, space after `//`  :p